src/pages/home/sidebar/OptionRowTitle/index.js

```diff
@@ -0,0 +1,136 @@
+     * @param {Number} index Used to get the Ref to the node at the current index.
+     * @returns {Number} Distance to shift the tooltip horizontally
+     * @memberof OptionRowTitle
+     */
+    getTooltipShiftX(index) {
+        // Only shift when containerLayout or Refs to text node is available .
+        if (!this.containerLayout || !this.cRefs[index]) {
+            return;
+        }
+        const {width: containerWidth, left: containerLeft} = this.containerLayout;
+
+        // We have to return the value as Number so we can't use `measureWindow` which takes a callback
+        const {width: textNodeWidth, left: textNodeLeft} = this.cRefs[index].getBoundingClientRect();
+        const tooltipX = (textNodeWidth / 2) + textNodeLeft;
+        const containerRight = containerWidth + containerLeft;
+        const textNodeRight = textNodeWidth + textNodeLeft;
+        const newToolX = textNodeLeft + ((containerRight - textNodeLeft) / 2);
+
+        // When text right end is beyond the Container Right end
+        return textNodeRight > containerRight ? -(tooltipX - newToolX) : 0;
+    }
+
+
+    render() {
+        const {
+            option, style, tooltipEnabled, numberOfLines,
+        } = this.props;
+
+        if (!tooltipEnabled) {
+            return <Text style={style} numberOfLines={numberOfLines}>{option.text}</Text>;
+        }
+        return (
+
+            // Tokenization of string only support 1 numberofLines on Web
+            <Text
+                style={[style, styles.optionDisplayNameTooltipWrapper]}
+                onLayout={this.setContainerLayout}
+                numberOfLines={1}
+                ref={this.setContainerRef}
+            >
+                {_.map(option.participantsList, (participant, index) => {
+                    // We need to get the refs to all the names which will be used to correct
+                    // the horizontal position of the tooltip
+                    const setChildRef = this.setOptionChildRef(index);
+                    return (
+                        <Fragment key={index}>
+                            <Tooltip
+                                key={index}
+                                text={participant.login}
+                                containerStyle={styles.dInline}
+                                shiftHorizontal={() => this.getTooltipShiftX(index)}
+                            >
+                                <Text ref={setChildRef}>
+                                    {participant.displayName}
+                                </Text>
+                            </Tooltip>
+                            {index < option.participantsList.length - 1
+                                ? <Text>,&nbsp;</Text>
+                                : null}
+                        </Fragment>
+                    );
+                })}
+                {
+                    option.participantsList.length > 1 && this.state.isEllipsisActive
+                        ? (
```

Comment:
Since the third piece of this ternary is just `null`, I think you can instead just do:

```jsx
option.participantsList.length > 1 && this.state.isEllipsisActive && (
    <View style={styles.optionDisplayNameTooltipEllipsis}>
         <Tooltip text={option.tooltipText}>
             {/* There is some Gap for real ellipsis so we are adding 4 `.` to cover */}
             <Text>....</Text>
         </Tooltip>
     </View>
)
```